### ISSUE TITLE
ebayui-core-react/EbayTextbox: Component textfield does not support accessibility

### ISSUE THREAD

**Issue by rahulebay** [eBay/evo-web#301]
### Description

In storybook preview page: https://opensource.ebay.com/evo-web/ebayui-core-react/main/?path=/story/form-input-ebay-textbox--default-textbox

Accessibility > Violations Tab:

### Screenshots

<img width="1530" height="762" alt="Image" src="https://github.com/user-attachments/assets/1e9d0e9f-e0cd-4022-b1b5-f6ad1cb8918a" />

### Engineering pre-requisites

- [ ] Supports dark mode.
- [ ] Supports responsive design.
- [ ] Tokens have been provided (or already exist).

**Comment by rahulebay** [comment]
I checked its not there for Placeholder. 
But it is present for many others like: Fluid, Invalid, Multi-line, Disabled, Password, Floating etc

<img width="1576" height="834" alt="Image" src="https://github.com/user-attachments/assets/bee19e04-9c39-4fd1-914d-3e8e61969f4b" />

<img width="1584" height="852" alt="Image" src="https://github.com/user-attachments/assets/f1d0db56-d33e-4c02-95ab-46681dfb3ee6" />
<img width="1583" height="848" alt="Image" src="https://github.com/user-attachments/assets/50720d78-d1f3-47c3-8595-3440b95c61c7" />
<img width="1579" height="825" alt="Image" src="https://github.com/user-attachments/assets/5e729d60-a6be-4294-8b2f-39ab51b8df53" />
